web/src/features/charts/BreakdownChart.tsx

```diff
@@ -51,6 +52,18 @@ function BreakdownChart({
   const { t } = useTranslation();
   const isHourly = useAtomValue(isHourlyAtom);
 
+  const [selectedData, setSelectedData] = useState<Record<string, boolean>>({});
+
+  // TODO(cady): is there a better way to reset if we switch between emissions & electricity panels?
+  useEffect(() => setSelectedData({}), [displayByEmissions]);
```

Comment:
Event listeners maybe?

Should be one for onButtonPress on the emissions and electricity buttons.

Reply:
You might have to use an atom instead of a state though, but I still think it would be preferable if possible.